src/MongoDB/ReadPreference.c

```diff
@@ -353,6 +436,7 @@ static HashTable* php_phongo_readpreference_get_properties_hash(zval* object, bo
 	HashTable*                   props;
 	const char*                  modeString = NULL;
 	const bson_t*                tags;
+	const bson_t*                hedge;
```

Comment:
Should `PHONGO_GET_PROPERTY_HASH_INIT_PROPS` use `4` instead of `3` below? I have some recollection of seeing that in an earlier version of this PR, but I could be confusing it with something else.

EDIT: Looks like I was confusing this with `array_init_size` in the `serialize` method.

Reply:
Ah, yes. Not sure if it makes a difference, but I updated it for the sake of consistency.

Reply:
IIRC, it just serves as a hint for the initial allocation. You can add more elements and PHP will re-size the HashTable as needed.